source/_integrations/fronius.markdown

```diff
@@ -32,6 +32,11 @@ resource:
   description: "The URL of the Fronius device (e.g., `http://192.0.2.0` or `http://fronius.local`)"
   required: true
   type: string
+scan_interval:
+  description: "Defines number of seconds for polling interval."
+  required: false
+  type: integer
+  default: "60"
```

Comment:
Scan interval is a generic platform option and not specific to this integration. Please revert this part.